components/automate-ui/src/app/page-components/insight-attributes-dropdown/insight-attributes-dropdown.component.html

```diff
@@ -0,0 +1,25 @@
+<div [@childAnim]>
+  <div class="bx--row">
+    <p class="instruction">Choose up to 5 attributes to display in the table:</p>
+  </div>
+  <ul class="bx--row">
```

Comment:
I was honestly just running into this today in another PR and realized there probably needs to be some sort of convention.  Its hard to pick since (in my opinion) its easier to figure out by looking at the html but significantly cleaner to extend in sass.

Either way - should be doing it all over.